tests/test_start_date.py

```diff
@@ -0,0 +1,149 @@
+import tap_tester.connections as connections
+import tap_tester.runner as runner
+from base import YotpoBaseTest
+from tap_tester.logger import LOGGER
+
+class YotpoStartDateTest(YotpoBaseTest):
+    """Instantiate start date according to the desired data set and run the test"""
+
+    start_date_1 = ""
```

Comment:
As we have only 1 function in the method, there is no meaning of defining variable and passing date in the function. One of it can be removed.